### DISCUSSION TITLE
voice conversion with XTTS_V2

### DISCUSSION THREAD

**Discussion by etlweather** [idiap/coqui-ai-TTS#75] [Q&A] [answered]
I could be completely wrong, but from what I read all over, I should be able to do voice-to-voice conversion with other models than `freevc24`.

I managed to make it work with `your_tts` by changing the command line parameters. Instead of using `source_wav` and `target_wav`, I used `reference_wav` and `speaker_wav` and added `language_idx`.

But this does not work with the `XTTS_V2` model.

What am I doing wrong? Or is this not even possible?

**Comment by eginhard** [comment, marked as answer]
No, this is not possible. XTTS only supports voice cloning, i.e. converting text to the voice of a given target speaker, but not voice-to-voice conversion. But we're planning to add 1-2 other voice conversion models soon.
